app/controllers/users/registrations_controller.rb

```diff
@@ -24,6 +20,7 @@ def create
       resource.transaction do
         # skip sending the Devise confirmation notification
         resource.skip_confirmation_notification!
+        set_invitation(resource)
```

Comment:
Using `set_invitation` here seems like overkill, since the User already loads the invitation token when saving. Is there a reason you prefer it over the pre-Devise way of setting it?
https://github.com/otwcode/otwarchive/blob/317ebf14ab1234cd9861e8f12214c593a0e44dfe/app/controllers/users_controller.rb#L145